cfme/control/explorer.py

```diff
@@ -619,6 +619,20 @@ def create(self, cancel=False):
             action=form_buttons.cancel if cancel else form_buttons.add)
         flash.assert_no_errors()
 
+    def copy(self, cancel=False):
+        """Copy this Policy from CFME.
+
+        Args:
+            cancel: Whether to cancel the process instead of copying.
+        """
+        sel.force_navigate(self.PREFIX + "policy",
+                           context={"policy_name": self.description})
+        cfg_btn(self.COPY_STRING, invokes_alert=True)
+        sel.handle_alert(cancel)
+        flash.assert_no_errors()
+        new_description = "Copy of {}".format(self.description)
+        return self.__class__(new_description)
```

Comment:
I would personally use `return type(self)(new_description)`. The `__` methods are not supposedto be called :)